.github/workflows/cypress-test-tenancy-disabled.yml

```diff
@@ -59,19 +59,22 @@ jobs:
 
       - name: Configure and Run OpenSearch Dashboards with Cypress Test Cases
         run: |
-          cd ./OpenSearch-Dashboards
-          echo 'server.host: "0.0.0.0"' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch.hosts: ["https://localhost:9200"]' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch.ssl.verificationMode: none' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch.username: "kibanaserver"' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch.password: "kibanaserver"' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch.requestHeadersWhitelist: [ authorization,securitytenant ]' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch_security.multitenancy.enabled: false' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch_security.readonly_mode.roles: ["kibana_read_only"]' >> ./config/opensearch_dashboards.yml
-          echo 'opensearch_security.cookie.secure: false' >> ./config/opensearch_dashboards.yml
-          nohup yarn start --no-base-path --no-watch &
+          cd ./opensearch-dashboards-${{ env.OPENSEARCH_VERSION}}
+          rm -rf ./config/opensearch_dashboards.yml
+          cat << 'EOT' > ./config/opensearch_dashboards.yml
+          server.host: "0.0.0.0"
+          opensearch.hosts: ["https://localhost:9200"]
+          opensearch.ssl.verificationMode: none
+          opensearch.username: "kibanaserver"
+          opensearch.password: "kibanaserver"
+          opensearch.requestHeadersWhitelist: [ authorization,securitytenant ]
+          opensearch_security.multitenancy.enabled: false
+          opensearch_security.readonly_mode.roles: ["kibana_read_only"]
+          opensearch_security.cookie.secure: false
+          EOT
+          nohup ./bin/opensearch-dashboards &
           sleep 500
```

Comment:
I think we should also remove the hardcoded sleep for here. Just like the one you did for oidc and saml workflow, we should catch the successful compiled msg in the log and proceed immediately

Reply:
Sure, that is a good suggestion

Reply:
Done in: [325f066](https://github.com/opensearch-project/security-dashboards-plugin/pull/1726/commits/325f066733164aef8027d379043acaeb514b2f1d)